src/Model/ItemList.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\Model;
+
+use Countable;
+use IteratorAggregate;
+use Traversable;
+use function count;
+
+final class ItemList implements Countable, IteratorAggregate
```

Comment:
generic name, could it be `Item(s)Page` given it contains a pointer to the next element?